torch/distributed/tensor/_ops/_pointwise_ops.py

```diff
@@ -686,7 +759,9 @@ def common_pointwise_strategy(
     register_op_strategy(op, schema_info=RuntimeSchemaInfo(static_kwargkey=["out"]))(
         pointwise_strategy
     )
-
+    # register_single_dim_strategy(
```

Comment:
purposefully not enabled yet. but I left the commented code here since I often want to switch it back on for testing